### PR TITLE
fix: add gas-limit flag

### PR DESCRIPTION
**Pull request by noot**

### CODE REVIEW

File: sequencer-relayer/src/da.rs [hunk 1]
```diff
@@ -11,8 +11,8 @@ use crate::base64_string::Base64String;
 use crate::sequencer_block::{IndexedTransaction, Namespace, SequencerBlock, DEFAULT_NAMESPACE};
 use crate::types::Header;
 
-static DEFAULT_PFD_FEE: i64 = 2_000;
-static DEFAULT_PFD_GAS_LIMIT: u64 = 90_000;
+pub const DEFAULT_PFD_GAS_LIMIT: u64 = 1_000_000;
```
Comment (steezeburger) [inline sequencer-relayer/src/da.rs:14]: Does this need to be pub?

Comment (noot) [reply sequencer-relayer/src/da.rs:14]: no, will change

**Review by steezeburger — APPROVED** [review]

File: sequencer-relayer/src/da.rs [hunk 2]
```diff
@@ -132,7 +133,16 @@ impl CelestiaClient {
...
             .wrap_err("failed creating celestia node client")?;
-        Ok(CelestiaClient { client: cnc })
+        Ok(CelestiaClient {
+            client: cnc,
+            gas_limit: DEFAULT_PFD_GAS_LIMIT,
+        })
+    }
+
+    /// sets the gas limit to be used for PFDs.
+    pub fn with_gas_limit(mut self, gas_limit: u64) -> Self {
```
[the remark this answers is not in the document]
Comment (noot) [reply sequencer-relayer/src/da.rs]: updated!

File: sequencer-relayer/src/da.rs [hunk 3]
```diff
@@ -11,8 +11,8 @@ use crate::base64_string::Base64String;
 use crate::sequencer_block::{IndexedTransaction, Namespace, SequencerBlock, DEFAULT_NAMESPACE};
 use crate::types::Header;
 
-static DEFAULT_PFD_FEE: i64 = 2_000;
-static DEFAULT_PFD_GAS_LIMIT: u64 = 90_000;
+const DEFAULT_PFD_GAS_LIMIT: u64 = 1_000_000;
```
[the remark this answers is not in the document]
Comment (noot) [reply sequencer-relayer/src/da.rs:14]: the original value was taken from celestia, but we ran into issues with it being too low. so yeah right now it's basically arbitrary, I can research celestia gas costs to make it less arbitrary

**Pull request merged**